packages/node_modules/@webex/webex-core/src/lib/services/service-url.js

```diff
@@ -75,15 +89,17 @@ const ServiceUrl = AmpState.extend({
    * @param {string} url
    * @returns {boolean}
    */
-  removeHost(url) {
+  failHost(url) {
```

Comment:
Could you update the jsdoc on this renamed function? It still references removing the host.